src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1771,7 +1773,7 @@ private void OpenWorkspace(WorkspaceModel ws)
             }
 
             AddWorkspace(ws);
-            CurrentWorkspace = ws;
+            //CurrentWorkspace = ws;
```

Comment:
Hope this doesn't cause any side-effects. As noted above I have moved this so that it happens a little later in the workspace opening sequence.